contents.json

```diff
@@ -606,6 +606,10 @@
     "title": "GraphQL",
     "id": "graphql",
     "parent": "data-management"
+  }, {
+    "title": "SOAP Protocol",
```

Comment:
@priore not sure about a need of mentioning "Protocol" in the title. Can we keep it only "SOAP" instead of  "SOAP Protocol" there?